### PR TITLE
#Added General and Professional Skills

### PR DESCRIPTION
**Pull request by usanaphtal112**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,21 @@
 
 1. [Programming Languages](Professional_Skills_0002/Programming_Languages_%232_0002.md)
 2. [Web Development](Professional_Skills_0002/Web_Development_%232_0002.md)
+
+## Naphtal 0003
+
+### General Skills
+
```
Comment (AbuuIbtisaam) [inline README.md:17]: Please, review and test your links (All links in this section) they are not redirecting to the intended page.

File: README.md [hunk 2]
```diff
@@ -10,3 +10,21 @@
...
+1. [Communication Skills](General_Skills_0003\Communication_%231_0003.md)
+
+2. [Negotiation Skills](General_Skills_0003\Negotiation_Skills_%231_0003.md)
+
+3. [Social Skills](General_Skills_0003\Social_Skills_%231_0003.md)
+
+### Professional Skills
+
+1. [Machine Learning](Professional_Skills_0003\Machine_Learning_Skills_%232_0003.md)
+
```
Comment (AbuuIbtisaam) [inline README.md:27]: Please, review and test your links (All links) they are not redirecting to the intended page.

File: README.md [hunk 3]
```diff
@@ -10,3 +10,21 @@
...
+### General Skills
+
+1. [Communication Skills](General_Skills_0003\Communication_%231_0003.md)
+
+2. [Negotiation Skills](General_Skills_0003\Negotiation_Skills_%231_0003.md)
+
+3. [Social Skills](General_Skills_0003\Social_Skills_%231_0003.md)
+
+### Professional Skills
+
```
Comment (AbuuIbtisaam) [inline README.md:25]: Please, review and test your links (All Links in this section) they are not redirecting to the intended page.

**Review by AbuuIbtisaam — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (usanaphtal112) [reply README.md:27]: Resolved!

**Review by AbuuIbtisaam — APPROVED** [review]

**Review by leandreAlly — APPROVED** [review]

**Pull request merged**